tests/providers/amazon/aws/executors/ecs/test_ecs_executor.py

```diff
@@ -367,6 +368,7 @@ def test_stopped_tasks(self):
 class TestAwsEcsExecutor:
     """Tests the AWS ECS Executor."""
 
+    @pytest.mark.skipif(not AIRFLOW_V_2_10_PLUS, reason="Test requires Airflow 2.10+")
```

Comment:
Hey @o-nikolas -> I skipped this failing tests for Airlfow < 2.10 - indeed it looks like calling `change_state` has been added in https://github.com/apache/airflow/pull/39212 and is only going to happen in Airflow 2.10+ . I am not sure what effect it has on AwsEcsExecutor- whether it will have any side effect when runnning on Airflow 2.9 - I guess the executor is still in "experimental" phase, but in case this change is somewhat braking for the Executor, it would be wise to add a runtime check for the upcoming Amazon provider's executor to fail explicitly for Airflow < 2.10

Reply:
Hey @potiuk, There is a guard in the code that will allow the executor to run just fine on pre 2.10 (the bugs that I fixed will still be there of course, but it will still operate). So no worries with that. But I didn't anticipate the tests being run against an older version so thanks for wrapping it in that skip! That should be the only action required here.

Reply:
Cool.